packages/editor-ui/src/components/WorkflowHistory/WorkflowHistoryListItem.vue

```diff
@@ -0,0 +1,178 @@
+<script setup lang="ts">
+import { computed, ref, onMounted } from 'vue';
+import dateformat from 'dateformat';
+import type { UserAction } from 'n8n-design-system';
+import type { WorkflowHistory, WorkflowHistoryActionTypes } from '@/types/workflowHistory';
+import type { TupleToUnion } from '@/utils/typeHelpers';
+import { useI18n } from '@/composables';
+
+const props = defineProps<{
+	item: WorkflowHistory;
+	index: number;
+	actions: UserAction[];
+	active: boolean;
+}>();
+const emit = defineEmits<{
+	(
+		event: 'action',
+		value: { action: TupleToUnion<WorkflowHistoryActionTypes>; id: WorkflowHistory['versionId'] },
+	): void;
+	(event: 'preview', value: { event: Event; id: WorkflowHistory['versionId'] }): void;
+	(event: 'mounted', value: { index: number; offsetTop: number; active: boolean }): void;
+}>();
+
+const i18n = useI18n();
+
+const actionsVisible = ref(false);
+const itemElement = ref<HTMLElement | null>(null);
+
+const formattedCreatedAtDate = computed<string>(() => {
+	const currentYear = new Date().getFullYear().toString();
+	const [date, time] = dateformat(
+		props.item.createdAt,
+		`${props.item.createdAt.startsWith(currentYear) ? '' : 'yyyy '} mmm d"#"HH:MM`,
+	).split('#');
+
+	return i18n.baseText('workflowHistory.item.createdAt', { interpolate: { date, time } });
+});
+
+const authors = computed<{ label: string; tooltip: string }>(() => {
+	const allAuthors = props.item.authors.split(', ');
```

Comment:
The contract is meant to be like this? Strikes me that `authors` should be `string[]` already.

Reply:
Yes, the backend sends it as a coma separated list